infrastructure/sandbox/PreProvisioner/lambda/deploy_terraform/fleet/templates/deployment.yaml

```diff
@@ -50,8 +50,6 @@ spec:
             memory: {{ .Values.resources.requests.memory }}
         env:
           ## BEGIN FLEET SECTION
-          - name: FLEET_DEMO
-            value: "1"
```

Comment:
made this change hoping to save you some time @fleetdm/infra , but lmk if this is not a good idea and I will revert.

Reply:
:accept: